src/Box/types.ts

```diff
@@ -0,0 +1,28 @@
+import { BaseSpacingProps } from '../BaseSpacing';
+
+type FlexAlignType =
+  | 'stretch'
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'baseline';
+type FlexDirectionType = 'row' | 'column';
+type FlexJustifyType =
+  | 'flex-start'
+  | 'flex-end'
+  | 'center'
+  | 'space-between'
+  | 'space-around'
+  | 'space-evenly';
+type FlexWrapType = 'nowrap' | 'wrap' | 'wrap-reverse';
+
+export interface BoxProps extends BaseSpacingProps {
+  alignItems?: FlexAlignType;
+  as?: React.ElementType | keyof JSX.IntrinsicElements;
+  children: JSX.Element | string;
+  className?: string;
+  display?: 'block' | 'flex';
+  flexDirection?: FlexDirectionType;
+  justifyContent?: FlexJustifyType;
+  wrap?: FlexWrapType;
```

Comment:
To follow the standard naming, `wrap` should be `flexWrap` instead?

Reply:
Changed!